docs/guides/deploy-hyperlane-local-agents.mdx

```diff
@@ -0,0 +1,397 @@
+import TerminologyPartial from "/docs/partials/deploy-hyperlane/_terminology.mdx";
+import SetupKeysPartial from "/docs/partials/deploy-hyperlane/_setup-keys.mdx";
+import DeployContractsPartial from "/docs/partials/deploy-hyperlane/_deploy-contracts.mdx";
+import SendTestMessagesPartial from "/docs/partials/deploy-hyperlane/_send-test-messages.mdx";
+import DeployWarpRoutePartial from "/docs/partials/deploy-hyperlane/_deploy-warp-route.mdx";
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Deploy Hyperlane with Local Agents
+
+:::tip
+
+This guide is for advanced users who have some understanding of configuring and deploying Hyperlane.
+
+If not done so already, beginners should get started by [**deploying Hyperlane with Kurtosis agents**](/docs/deploy-hyperlane).
+
+:::
+
+<TerminologyPartial />
+
+## Overview
+
+There are six steps in this guide:
+
+1. [<b>Set up keys</b>](#1-set-up-keys) that you will use to deploy contracts and run validators and relayers.
+1. [<b>Deploy contracts</b>](#2-deploy-contracts) to the local chain and to every remote chain with which the local chain will be able to send and receive messages.
+1. [<b>Run a validator</b>](#3-run-a-validator) to provide the signatures needed for the Interchain Security Modules you deployed.
+1. [<b>Run a relayer</b>](#4-run-a-relayer) to send and receive messages between the chains you deployed contracts to.
+1. [<b>Send a test message</b>](#5-send-test-messages) to confirm that your relayer is able to deliver messages to and from each pair of chains.
+1. [<b>Deploy a Warp Route</b>](#6-optional-deploy-a-warp-route) to send token value, not just messages, across chains.
+
+## Getting Started
+
+## 1. Set up keys
+
+<SetupKeysPartial />
+
+## 2. Deploy contracts
+
+<DeployContractsPartial />
+
+## 3. Run a validator
+
+Validators provide the security for messages sent _from_ your chain to remote chains.
```

Comment:
I like this, let's also add that they're only required when using a Multisig ISM and link to https://docs.hyperlane.xyz/docs/protocol/ISM/multisig-ISM